extensions/wikia/Privacy/RemovalAuditLog.php

```diff
@@ -10,36 +10,74 @@ class RemovalAuditLog {
 
 	public static function createLog( $userId ) {
 		$db = self::getDb( DB_MASTER );
-		$db->insert( self::LOG_TABLE, ['user_id' => $userId], __METHOD__ );
+		$db->insert( self::LOG_TABLE, [ 'user_id' => $userId, 'global_data_removed' => false ], __METHOD__ );
+
 		return $db->insertId();
 	}
 
+	/**
+	 * Mark that global user data was removed for the given log id.
+	 *
+	 * @param $logId
+	 */
+	public static function markGlobalDataRemoved( $logId ) {
+		self::getDb( DB_MASTER )
+			->update( self::LOG_TABLE, [ 'global_data_removed' => true ], [ 'log_id' => $logId ], __METHOD__ );
+	}
+
+	public static function allWikiDataWasRemoved( $logId, $detailsDbType = DB_MASTER ) {
```

Comment:
Do we need to have a default here? Can't we always explicitly use one? Or is this some kind of a style in MW?